### PR TITLE
GSdx: Remove GLX in favor of EGL

### PR DESCRIPTION
**Pull request by JibbityJobbity**
As title suggests, removes loading OpenGL using GLX in favor of EGL which is newer and supports Wayland.

### CODE REVIEW

**Comment by orbea** [comment]
Is removing it really the best idea? How about just changing the default?

**Comment by tadanokojin** [comment]
I had jib make this PR assuming we might remove it. But it's just for testing right now to see how well things go.

**Comment by kenshen112** [comment]
[https://imgur.com/a/pisjAGQ](Specs I tested this pr with) There's no noticeable change in performance once we figured out the cmake issue jib was having and got gsdx to compile correctly

**Comment by JibbityJobbity** [comment]
I have an Nvidia GTX 980 ti with driver version `450.57`, would be nice to have someone testing AMD on Ubuntu 20.04.

**Comment by kimberly990** [comment]
Works fine for me. AMD RX 570 on Mesa 20.1.3-1, tested on Manjaro KDE 20.0.3

File: cmake/SearchForStuff.cmake [hunk 1]
```diff
@@ -87,6 +87,8 @@ endif()
 ## Use CheckLib package to find module
 include(CheckLib)
 if(Linux)
+    check_lib(EGL EGL EGL/egl.h)
+    check_lib(X11_XCB X11-xcb X11/Xlib-xcb.h)
```
Comment (turtleli) [inline cmake/SearchForStuff.cmake:91]: EGL isn't limited to Linux.

**Pull request merged**